### DISCUSSION TITLE
[V3] Input gets not resets

### DISCUSSION THREAD

**Discussion by hebbet** [livewire/livewire#6045] [7. Bugs (archived)]
### Livewire version

3.0-beta4

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

when using that example of data-binding the todo input doesn't get reset with beta 3. works fine with v2

https://livewire.laravel.com/docs/properties#data-binding

**Comment by gehrisandro** [comment]
Probably you've called `reset` without any parameters which behaves different in v3 than v2.

This [PR](https://github.com/livewire/livewire/pull/6118) brings the behaviour from v2 back. Lets have a look what @calebporzio thinks.

**Reply by hebbet** [reply]
not even  $this->todo = ''; works

**Reply by gehrisandro** [reply]
`$this->todo = ''` and `$this->reset('todo');` both work for me (without my PR).

**Reply by hebbet** [reply]
for me with above mentioned version it doesn't work.

that is why i opened that discussion here.

**Reply by gehrisandro** [reply]
Could you please retry with the latest version (3.0-beta5)?

If it still fails, please provide your full component. Maybe I can help you to spot the issue.

**Comment by hebbet** [comment]
didn't notice that info:

https://livewire.laravel.com/docs/upgrading#alpinejs:~:text=%22Livewire%20V3%20(beta)%20with%20Laravel%20Breeze%20or%20Laravel%20Jetstream%22
